src/main/java/com/glencoesoftware/bioformats2raw/IProgressListener.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2022 Glencoe Software, Inc. All rights reserved.
+ *
+ * This software is distributed under the terms described by the LICENSE.txt
+ * file you can find at the root of the distribution bundle.  If the file is
+ * missing please request a copy by contacting info@glencoesoftware.com
+ */
+package com.glencoesoftware.bioformats2raw;
+
+import java.util.EventListener;
+
+public interface IProgressListener extends EventListener {
+
+  /**
+   * Indicates the beginning of processing a particular resolution.
+   *
+   * @param series the series index being processed
+   * @param resolution the resolution index being processed
+   * @param tileCount the total number of tiles in this resolution
+   */
+  void notifyResolution(int series, int resolution, int tileCount);
```

Comment:
My immediate feedback is that the contract of the notification events were not immediately clear just based on their names.  thought `notifyDone` would be a single event at the end of the conversion and I was confused by the fact `notifyResolution` indicates the start of the processing for a given resolution while `notifyChunk` indicates the end of the processing for a chunk.

A proposal to address this would be to name these methods e.g. as `notify[Series|Resolution|Chunk][Start|End]`. Not all these need to be defined initially but that would allow more granularity to be added in the future.